pkg/cmd/ui/ui.go

```diff
@@ -86,40 +85,30 @@ func EnterImageRepoExternalRepository() string {
 }
 
 // EnterOutputPath allows the user to specify the path where the gitops configuration must reside locally in a UI prompt.
-func EnterOutputPath() string {
-	var outputPath string
+func EnterOutputPath(originalPath string) string {
+	var outputPath = originalPath
 	prompt := &survey.Input{
 		Message: "Provide a path to write GitOps resources?",
 		Help:    "This is the path where the GitOps repository configuration is stored locally before you push it to the repository GitopsRepoURL",
 		Default: ".",
 	}
-
-	err := survey.AskOne(prompt, &outputPath, nil)
-	exists, filePathError := ioutils.IsExisting(ioutils.NewFilesystem(), filepath.Join(outputPath, "pipelines.yaml"))
-	if exists {
-		SelectOptionOverwrite(outputPath)
-	}
-	if !exists {
-		err = filePathError
+	for true {
+		exists, err := ioutils.IsExisting(ioutils.NewFilesystem(), filepath.Join(outputPath, "pipelines.yaml"))
+		if exists {
+			doOverwrite := SelectOptionOverwrite()
+			if doOverwrite == "no" {
+				handleError(survey.AskOne(prompt, &outputPath, nil))
+			} else {
+				break
+			}
+		} else {
+			break
+		}
+		handleError(err)
```

Comment:
Ok, makes sense. I now need to pass the current value of the `overwrite` flag so that you won't get prompted if the user actually specified the `--overwrite` option. Look at how streamlined it is now:

```
kam bootstrap --gitops-repo-url https://github.com/keithchong/gitops9.git --service-repo-url https://github.com/keithchong/taxi.git --overwrite=true
  
Checking dependencies

 ✓  Checking if Sealed Secrets is installed with the default configuration [158ms]
 ✓  Checking if ArgoCD Operator is installed with the default configuration [348ms]
 ✓  Checking if OpenShift Pipelines Operator is installed with the default configuration [413ms]
  
Starting interactive prompt

? Please provide a token used to authenticate requests to "https://github.com/keithchong/taxi.git" [? for help] ****************************************
  
Completing Bootstrap process

 ✓  Options used:
    Service repository: https://github.com/keithchong/taxi.git
    GitOps repository: https://github.com/keithchong/gitops9.git
    Image repository: image-registry.openshift-image-registry.svc:5000/cicd/taxi
    Commit status tracker: true
    Output folder: .
    Overwrite output folder: true
  
 ✓  Authentication tokens encrypted in secrets
 ✓  Pipelines tracker has been configured
 ✓  OpenShift Pipelines resources created
 ✓  Openshift Route for EventListener created
 ✓  Created dev, stage and CICD environments
 ✓  Bootstrapped OpenShift resources successfully

 Next Steps:
 Please refer to https://github.com/redhat-developer/kam/tree/master/docs to get started.

```